VS/VehicleSignals.rdf

```diff
@@ -1226,6 +1230,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;CushionPositionSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Cushion Position Sensor</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.